_includes/delete-payment-order-token.md

```diff
@@ -0,0 +1,49 @@
+{% capture api_resource %}{% include api-resource.md %}{% endcapture %}
+{% assign token_field_name = include.token_field_name %}
+{% capture token_url %}
+    /psp/{{ api_resource }}/{{ token_field_name }}s/{{- page.payment_token -}}
+{% endcapture %}
+{% assign token_url=token_url | strip %}
+
+## Delete {{ token_field_name }}
+
+Payers should be able to delete payment tokens that are associated to
```

Comment:
Should this text be differentiated from delete_token? 
We could potentially migrate all delete paymentOrder recurrenceTokens to use this file